### ISSUE TITLE
Any plans to upgrade the package for pydantic v2?

### ISSUE THREAD

**Issue by kodzonko** [Phil997/pydantic-sqlite#9]
Hi, are there any plans to upgrade the package for pydantic v2?

I didn't test if it actually doesn't work with the most recent version in current config poetry doesn't resolve the dependency as pydantic-sqlite depends on `pydantic<2.0.0`.

output:
```
$ poetry add pydantic-sqlite                                  
Using version ^0.2.1 for pydantic-sqlite

Updating dependencies
Resolving dependencies...

Because no versions of pydantic-sqlite match >0.2.1,<0.3.0
 and pydantic-sqlite (0.2.1) depends on pydantic (>=1.8.2,<2.0.0), pydantic-sqlite (>=0.2.1,<0.3.0) requires pydantic (>=1.8.2,<2.0.0).
So, because cinema-repertoire-analyzer depends on both pydantic (^2.5.3) and pydantic-sqlite (^0.2.1), version solving failed.
```

@Phil997 in case you don't plan to upgrade it yourself are you open to accept a PR that upgrades this library to a new pydantic version? I might be be able to contribute at some point.

**Comment by PJ-Schulz** [comment]
Hello @kodzonko,

I am currently very busy with work and other projects, so I don't have time to update pydantic 2. At work I also use pydantic 2 and yes, this package should be updated.

Feel free to open a Pull Request and update the dependency.

**Comment by Phil997** [comment]
Hello @kodzonko,

I am currently very busy with work and other projects, so I don't have time to update pydantic 2. At work I also use pydantic 2 and yes, this package should be updated.

Feel free to open a Pull Request and update the dependency.

**Comment by Phil997** [comment]
Done in #16 and available with verison 0.3.0
